stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -154,22 +152,22 @@ internal class ChatDomainImpl internal constructor(
      * The total unread message count for the current user.
      * Depending on your app you'll want to show this or the channelUnreadCount
      */
-    override val totalUnreadCount: LiveData<Int> = _totalUnreadCount
+    override val totalUnreadCount: LiveData<Int> = _totalUnreadCount.asLiveData()
 
     /**
      * the number of unread channels for the current user
      */
-    override val channelUnreadCount: LiveData<Int> = _channelUnreadCount
+    override val channelUnreadCount: LiveData<Int> = _channelUnreadCount.asLiveData()
 
     /**
      * list of users that you've muted
      */
-    override val muted: LiveData<List<Mute>> = _mutedUsers
+    override val muted: LiveData<List<Mute>> = _mutedUsers.asLiveData()
 
     /**
      * if the current user is banned or not
      */
-    override val banned: LiveData<Boolean> = _banned
+    override val banned: LiveData<Boolean> = _banned.asLiveData()
```

Comment:
In the future would be great to provide it as an extra artifact that provides extensions to work with LiveData, and remove LiveData as a dependency of our module